infra.bs

```diff
@@ -97,6 +97,11 @@ violation.
 either width or height is zero"), means an inclusive "or" (implying "or both"), unless it is called
 out as being exclusive (with "but not both").
 
+<p>A <dfn export>user agent</dfn> is any software that acts on behalf of a user, for example by
```

Comment:
```suggestion
<p>A <dfn export>user agent</dfn> is any software that acts on behalf of a user; for example by
```

Reply:
The interwebs say to use a semicolon here (and a comma after) when the second bit is a complete sentence, which it isn't here. I don't feel strongly, of course.

Reply:
> The interwebs say to use a semicolon here (and a comma after) when the second bit is a complete sentence, which it isn't here. I don't feel strongly, of course.

I don’t feel strongly about it either — so it’s fine by me to leave it as-is

Reply:
I still think s/for example/such as/ is a better approach.